.mergify.yml

```diff
@@ -0,0 +1,30 @@
+pull_request_rules:
+  - name: automatic merge when CI passes and 2 reviews
+    conditions:
+      - "#approved-reviews-by>=2"
+      - "#review-requested=0"
+      - "#changes-requested-reviews-by=0"
+      - "#commented-reviews-by=0"
+      - status-success=continuous-integration/travis-ci/pr
+      - status-success=default
+      - status-success=codecov/project
+      - status-success=codecov/patch
+      - base=dev
+      - label!=dot_not_merge
+      - label!=wip
+      - label!=waiting
+      - label!=merge_after_release
+      - author=@CoreTeam
```

Comment:
```suggestion
      - author=@coreteam
```

Reply:
this  should be the id, not the name (tested in https://github.com/CanalTP/transit_model)